tests/unit_test/analytics/analytics_processor_test.py

```
@@ -0,0 +1,240 @@
+        }
+
+        with pytest.raises(AppException):
+            AnalyticsPipelineProcessor.update_scheduled_task(str(obj.id), "b1", "u1", payload)
+
+    def test_get_pipeline_code(self):
+        data = {
+            "bot": "b1",
+            "name": "test_name",
+            "pyscript_code": "print('Hello, World!')",
+        }
+        result = CallbackConfig.create_entry(**data)
```

Ruff (0.14.7)

Local variable `result` is assigned to but never used

Remove assignment to unused variable `result`

(F841)